base/collection.go

```diff
@@ -176,6 +124,68 @@ func GetGocbV2BucketFromCluster(cluster *gocb.Cluster, spec BucketSpec, waitUnti
 	return gocbv2Bucket, nil
 }
 
+// GetGocbCluster opens a connection to the Couchbase cluster and returns a *gocb.Cluster for the specified BucketSpec.
+func GetGocbCluster(spec BucketSpec) (*gocb.Cluster, error) {
```

Comment:
possible fix: passing a context into this for multiple dbs might be nice for logging